spec/mailers/provider_mailer_spec.rb

```diff
@@ -39,20 +39,20 @@
                                  ))
     @provider_user = @application_choice.provider.provider_users.first
     @provider_user.update(first_name: 'Johny', last_name: 'English')
-    @application_choice.application_form.update(submitted_at: Time.zone.now - 5.days)
+    @application_choice.application_form.update(submitted_at: Time.zone.now - 5.days, support_reference: '123A')
```

Comment:
I was wondering if it would be better to define the support_reference when we are creating the application form on lines 35-39.
```
create(
  :completed_application_form,
   first_name: 'Harry',
   last_name: 'Potter',
   support_reference: '123A',
 ))
```

Reply:
Yes, much better, thanks